resources/sass/abstracts/_mixins.scss

```diff
@@ -0,0 +1,90 @@
+@mixin modal-content {
+    margin-left: 18% !important;
```

Comment:
Refrain from using important. Why did you face such a need ?

Reply:
removed all the important from scss files.

Reply:
done